delta/plugins/storage/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/disk/DiskStorageFetchFileSpec.scala

```diff
@@ -30,6 +31,14 @@ class DiskStorageFetchFileSpec
       consume(source) shouldEqual "file content"
       Files.delete(file)
     }
-  }
 
+    "deal with a missing file" in {
+      val volume       = Files.createTempDirectory("disk-access")
+      val file         = volume.resolve("my/file.txt")
+      Files.createDirectories(file.getParent)
+      val fullFilePath = file.toString
+
+      DiskStorageFetchFile(Uri.Path(fullFilePath)).rejected shouldEqual FetchFileRejection.FileNotFound(fullFilePath)
+    }
+  }
```

Comment:
I would have converted this test to MUnit, however it's using an Akka trait and it didn't seem worth it for this story